packages/langium/src/test/langium-test.ts

```diff
@@ -380,3 +381,18 @@ export function expectSemanticToken(tokensWithRanges: DecodedSemanticTokensWithR
     });
     expect(result).toHaveLength(1);
 }
+
+export interface TestSignatureHelpOptions {
+    options: SignatureHelpOptions[];
+    mergedOptions: SignatureHelpOptions;
+}
+
+export function expectMergeSignatureHelpOptions(testOptions: TestSignatureHelpOptions): void {
+    const mergedSignatureHelp = mergeSignatureHelpOptions(testOptions.options);
+
+    expect(mergedSignatureHelp?.triggerCharacters?.length).toEqual(testOptions.mergedOptions.triggerCharacters?.length);
+    expect(mergedSignatureHelp?.retriggerCharacters?.length).toEqual(testOptions.mergedOptions.retriggerCharacters?.length);
+
+    expect(mergedSignatureHelp?.triggerCharacters?.sort()).toEqual(testOptions.mergedOptions.triggerCharacters?.sort());
+    expect(mergedSignatureHelp?.retriggerCharacters?.sort()).toEqual(testOptions.mergedOptions.retriggerCharacters?.sort());
+}
```

Comment:
You can move this function and the associated interface to the test file, since we don't expect users to actually test their signature help options. We only need it to test our own `mergeSignatureHelpOptions` implementation.